printer_mods/henrikssn/euler_extruder/README.md

```diff
@@ -0,0 +1,49 @@
+# Euler Extruder for Voron AfterBurner
+
+### Overview
+
+The Euler Extruder is the most lightweight extruder option for the Voron AfterBurner, coming in at just 139 grams (including all hardware). There is a mount available for Nema17 based toolhead boards, such as [TurboCAN] and [Huvud], or you can screw any of the IGUS/Generic drag chain mount there.
+
+Performance-wise it is the same as Sherpa Mini, which is very similar to Orbiter/Galileo. All these are more performant than original Clockwork.
```

Comment:
Do you have some data to back up the claim `All these are more performant than original Clockwork` ?  It's one thing to say it is objectively lighter, but you can't say it's more performant without some data to show that.

Reply:
Added a reference

Reply:
There's no reference in the README, the link is empty.